detection-rules/asr_new_sender_or_reply_to_with_new_linked_dom.yml

```
@@ -0,0 +1,32 @@
+name: "New Sender or Reply-To with New Linked Domain"
```

"New Sender" is ambiguous here, is it prevalence new or newly registered
```suggestion
name: "Newly Registered Sender or Reply-To Domain with Newly Registered Linked Domain"
```
I don't like this either, but it's less ambiguous